frontend/pages/queries/edit/EditQueryPage.tsx

```diff
@@ -277,6 +282,10 @@ const EditQueryPage = ({
       console.error(updateError);
       if (updateError.data.errors[0].reason.includes("Duplicate")) {
         renderFlash("error", "A query with this name already exists.");
+      } else if (
+        updateError.data.errors[0].reason.includes(INVALID_PLATFORMS_REASON)
```

Comment:
Just a heads up, Sarah made a handy utility function in `interfaces/errors.ts` called `getErrorReason` which can get the error message for you. I've been trying to use it more and think we can use it here too.